src/app/app.component.ts

```diff
@@ -4,10 +4,12 @@ import { IArticle } from './interfaces/article.interface';
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.sass']
+  styleUrls: ['./app.component.sass'],
 })
 export class AppComponent {
-
+getId(title: string) {
```

Comment:
public identifier and return type missing, also indentation (formatting) and order wrong => functions go below the constructor and attributes and getters setters (this is not a getter)) go above the constructor